src/components/Bomberman.tsx

```diff
@@ -1,19 +1,25 @@
 import React = require('react');
 import * as BombermanMap from './BombermanMap';
+import styled from '../../node_modules/react-emotion';

```

Comment:
`import styled from 'react-emotion';`; you never need `node_modules` in the import path.